pkg/compare/compare.go

```diff
@@ -555,18 +555,18 @@ func newSummary(reference *Reference, c *MetricsCorelatorDecorator, numDiffCRs i
 func (s Summary) String() string {
 	t := `
 Summary
-CRs with diffs: {{ .NumDiffCRs }}
+Diff count: {{ .NumDiffCRs }}
```

Comment:
I am a little scared that this may get confused with the number of lines with diffs, or number of diffs where each file can contain a number of diffs. its important that its clear that this i the number of cluster CRS that have diffs when compare to reference crs, (the important part is that its clear that its counting the number of CRS).
so i think we should make this part clear.